storefront/pages/order/[id].tsx

```diff
@@ -0,0 +1,74 @@
+import { useRouter } from 'next/router';
+
+import Footer from '../../modules/layout/components/Footer';
+import getProductMediaUrl from '../../modules/products/utils/getProductMediaUrl';
+import Header from '../../modules/layout/components/Header';
+import useOrderDetailQuery from '../../modules/orders/hooks/useOrderDetailQuery';
+
+const OrderDetail = () => {
+  const router = useRouter();
+
+  const { order, loading } = useOrderDetailQuery({ orderId: router.query.id });
+  if (loading) return <p>loading .... </p>;
+  console.log(order.items);
+
+  return (
```

Comment:
Reuse as much as possible from here: 

https://github.com/unchainedshop/currybag-website/blob/master/storefront/pages/warenkorb.tsx

Reply:
And here: https://github.com/unchainedshop/currybag-website/blob/master/storefront/modules/cart/components/ManageCart.tsx